src/core/server/logging/layouts/conversions/timestamp.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import moment from 'moment-timezone';
+import { last } from 'lodash';
+
+import { Conversion } from './type';
+import { LogRecord } from '../../log_record';
+
+const timestampRegExp = /{timestamp({(?<format>[^}]+)})?({(?<timezone>[^}]+)})?}/gi;
```

Comment:
I think maybe my last comment here was missed: https://github.com/elastic/kibana/issues/41956#issuecomment-583093353

Is there a reason we can't support the same syntax as log4j? Eg. `%timestamp{options}` instead of `{timestamp{options}}`?

Reply:
my bad, I forgot to answer: agree, we can switch to log4h-like syntax, but I will do it in a separate PR since it affects all the tests.